addons/block_code/ui/blocks/control_block/control_block.gd

```diff
@@ -148,9 +143,18 @@ func format():
 		snap_container.custom_minimum_size.x = 30
 		snap_container.custom_minimum_size.y = 30
 		snap_container.size_flags_horizontal = Control.SIZE_SHRINK_BEGIN
-		snap_container.add_theme_constant_override("margin_left", Constants.CONTROL_MARGIN)
+
+		var snap_gutter := Control.new()
+		snap_gutter.name = "Background"
+		snap_gutter.set_script(preload("res://addons/block_code/ui/blocks/utilities/background/gutter.gd"))
```

Comment:
I've seen this in a couple places in here. Why not just instantiate the class directly?

Reply:
I'm not sure _why_ we don't do that, indeed. I'll fix that :)

Reply:
Yep, `set_script()` only makes sense in the BlockCode node when it replaces the parent script. All others should be fixed.